shell.core/src/main/java/org/metaborg/spoofax/shell/client/Repl.java

```diff
@@ -47,6 +37,7 @@
         this.display = display;
         this.invoker = invoker;
 
+        // TODO: this does not really belong here.
```

Comment:
@gfokkema and then an IEditor would use these methods internally somewhere in their methods, right?

Sounds good to me. Then each subclass can override if needed (but rather not, to keep a uniform look)